src/components/ui/DetailComponent.vue

```diff
@@ -0,0 +1,57 @@
+<script setup lang="ts">
+import { usePointsStore } from '@/stores/points'
+import { UiButtonWithTooltip } from '@sigrennesmetropole/cooperation_jn_common_ui'
+import { computed } from 'vue'
+
+const pointStore = usePointsStore()
+
+const ConformityLabel = computed(() => {
+  if (pointStore.conformity == 'conform') {
+    return '/src/assets/icons/standard.svg'
```

Comment:
It may be better to use it with import. E.g. `import standartIcon from '@/assets/icons/standard.svg'`. It will help with refactoring later (if needed).